### PR TITLE
fix(bcdb): search and replace methods

### PR DESCRIPTION
**Pull request by bearmit**
Cell values were not maching vocabulary, Dylan helped me. We added a new vocab "sans".

### CODE REVIEW

File: scripts/typescript/src/bcdb/bcdb.ts [hunk 1]
```diff
@@ -208,26 +208,19 @@ export class BCDBLoader {
...
             unit: null,          
           });
 
-          // can be: "saxs", "tem", or "rheology"
-          let phase_method = row[Column.phase_method]?.toLowerCase();
-
-          // The custom vocabulary does not allow "rheology"
-          // Replacing with "rheometer".
-          if(phase_method === "rheology")  phase_method = "rheometer";
-
```
Comment (bearmit) [inline scripts/typescript/src/bcdb/bcdb.ts:223]: I removed this because the XLSX now contains valid and lower case values.

**Comment by nh916** [comment]
> We added a new vocab "sans"

is this also added on the DB Schema and database?

We need to be sure every vocabulary word we add currently is replicated across all places needed so we don't encounter any errors

File: scripts/typescript/src/bcdb/bcdb.ts (hunk 1 above)
Comment (nh916) [reply scripts/typescript/src/bcdb/bcdb.ts:223]: nice! cleaning up code as you got! love it!

**Review by nh916 — APPROVED** [review]
I don't know this code base too well, but from what I can see it looks fine

**Comment by bearmit** [comment]
> > We added a new vocab "sans"
> 
> is this also added on the DB Schema and database?
> 
> We need to be sure every vocabulary word we add currently is replicated across all places needed so we don't encounter any errors

A pull request is pending (accepted by Dylan, waiting for Brilant to merge and to deploy). In order to generate the final JSON I have to wait for it. But this PR does not contain the final file, only the code.
So I'll merge this now.

**Pull request merged**